src/markdown-pages/docs/ecosystem/ecosystem-installing.md

```diff
@@ -0,0 +1,84 @@
+---
+path: "/docs/ecosystem/installing"
+title: "Installing the Galasa Ecosystem in Docker"
+---
+
+The following section explains how to bring up a Galasa Ecosystem on a Docker server by using a Docker Operator. Currently the Docker Operator only supports the amd64 platform. A s390x (zLinux) implementation is planned for a future release.
+
+The instructions assume a basic understanding of how Docker works.
+
+## Before you begin
+
+To deploy a Galasa ecosystem using the Docker Operator, you need a Docker Engine, preferably on a Linux server, but it can be run on a laptop. If you are using a Mac or Windows OS, you  need Docker Desktop running a "socat" container.
+
+The ecosystem requires a minimum of 4GB of memory to run everything, although 4GB is very tight, allowing a maximum of two parallel running tests.
+
+The VM requires a minimum of 8GB of memory, 4 VCPUs and 80GB of storage. Docker has a default value of 4GB of RAM set, so you need to change this value. In Docker desktop you do this by using the Preferences > Resources tab.
+
+We are not aware of a minimum version of Docker Engine the Docker Operator requires.
+
+If the server or workstation has a firewall running, you might need to open the following ports:-
+
+    2379 - etcd
+    5984 - couchdb
+    8080 - API
+    8081 - Nexus
+    8082 - Jenkins
+
+### Installing a Galasa ecosystem in a Docker Engine
+
+Due to the way the various servers communicate, the ecosystem needs to know the hostname or IP address of the server or workstation the Docker Engine is running on. Due to the way Docker works, you cannot use 127.0.0.1 or localhost, you need to use the actual workstation or server hostname or IP address. This is provided in a config.yaml file.
+
+The Docker Operator is  distributed from the 0.10.0-SNAPSHOT, an unreleased version of Galasa, Therefore, certain versions of the Galasa ecosystem servers must be provided in the config.yaml file. Copy the following YAML and create a config.yaml file on your server or workstation, making a note of the full path of the file:
+
+```
+hostname: {hostname}
+galasaRegistry: docker.galasa.dev
+version: 0.9.0
+engineController:
+  controllerVersion: 0.10.0-SNAPSHOT
+  engineVersion: 0.9.0
+```
+
+Change the {hostname} value to your hostname. Note the two spaces on the last two lines -  they are important in YAML.
+
+To deploy the Galasa ecosystem, issue the following Docker command:
+
+```
+docker run -it -v /var/run/docker.sock:/var/run/docker.sock -v {path}/config.yaml:/config.yaml docker.galasa.dev/galasa-docker-operator-amd64:0.9.0
+```
+where {path} is the full pathname to the directory containing your config.yaml file.
+
+When the command is complete all 9 docker containers should be running. Use the command ```docker ps``` to list the active containers.
+
+
+### Verifying the installation
+
```

Comment:
Log onto <hostname>:8082 and log onto the jenkins node using user: admin and password: galasaadmin and run the SIMBANK_IVT test that should be pre-configured

Then connect your IDE to the ecosystem, by pointing the plugins at the ecosystem's bootstrap e.g. `http://localhost:8080/bootstrap`

further thigns to do, deploy maven artifacts to the nexus

we will need to create some instructions for people not using the provided nexus to reconfigure the CPS to look for a different repo